docs/server-worker-extra-args

```diff
@@ -0,0 +1,18 @@
+Add the following to you main.tf and modify the extra arguements to your liking;   
```

Comment:
Can this file be formatted with Markdown? To do that, change the filename to have a `.md` at the end and include a title and wrap your example with code previews. Take a look at my other documents if you need an example or follow [Github's guide on Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).